lib/ruby_smb/dcerpc/netlogon/netr_server_req_challenge_request.rb

```diff
@@ -0,0 +1,25 @@
+require 'ruby_smb/dcerpc/ndr'
+
+module RubySMB
+  module Dcerpc
+    module Netlogon
+
+      # [3.5.4.4.1 NetrServerReqChallenge (Opnum 4)](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-nrpc/5ad9db9f-7441-4ce5-8c7b-7b771e243d32)
+      class NetrServerReqChallengeRequest < BinData::Record
+        attr_reader :opnum
+
+        endian :little
+
+        ndr_lp_str          :primary_name
```

Comment:
Same comment about using names that match the documentation:
```suggestion
        logon_srv_handle    :primary_name
```